### PR TITLE
Fix formatting issues in Actions Page

### PR DESCRIPTION
**Pull request by ashanthamara**
### Purpose
This PR will update following.
1. Reduced the Action page subtitle length.
2. Reduced the Pre Issue Acess Token page subtitle length
3. Improved loading place holders of actions page.
4. Add documentation links.
5. Improved update button loading state when activating/deactivating action
6. Improved side panel item categorization in the side panel.

### Related Issue:
- https://github.com/wso2/product-is/issues/20751

### CODE REVIEW

File: features/admin.actions.v1/pages/actions.tsx [hunk 1]
```diff
@@ -265,6 +266,51 @@ export const ActionTypesListingPage: FunctionComponent<ActionTypesListingPageInt
             } ];
     };
 
+
```
Comment (Achintha444) [inline features/admin.actions.v1/pages/actions.tsx:269]:
```suggestion
```

File: features/admin.actions.v1/pages/actions.tsx [hunk 2]
```diff
@@ -265,6 +266,51 @@ export const ActionTypesListingPage: FunctionComponent<ActionTypesListingPageInt
...
+        const placeholders: ReactElement[] = [];
+        const cardsPerRow: number[] = [ 3, 1 ];
+
+        for (let rowIndex: number = 0; rowIndex < cardsPerRow.length; rowIndex++) {
+            const cardsInRow: number = cardsPerRow[ rowIndex ];
+            const cards: ReactElement[] = [];
+
+            for (let columnIndex: number = 0; columnIndex < cardsInRow; columnIndex++) {
+                cards.push(
+                    <div data-componentid={ `${ _componentId }-loading-card` }>
```
Comment (Achintha444) [inline features/admin.actions.v1/pages/actions.tsx:283]: why do we need this div, can't we directly use the Box for this ?

Comment (ashanthamara) [reply features/admin.actions.v1/pages/actions.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/6764/commits/cf1bd171b56be2d259b3fe7dbf5679164eb9a7cf

File: features/admin.actions.v1/pages/actions.tsx (hunk 1 above)
Comment (ashanthamara) [reply features/admin.actions.v1/pages/actions.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/6764/commits/cf1bd171b56be2d259b3fe7dbf5679164eb9a7cf

**Review by brionmario — APPROVED** [review]

File: features/admin.actions.v1/pages/actions.tsx [hunk 3]
```diff
@@ -275,36 +321,18 @@ export const ActionTypesListingPage: FunctionComponent<ActionTypesListingPageInt
...
-                            >
+            { isActionTypesConfigsLoading ? renderLoadingPlaceholder() : (
+                <div className="action-types-grid-wrapper" data-componentid={ `${ _componentId }-grid` }>
+                    <div className="action-types-grid">
+                        { actionTypesCardsInfo().map((cardProps: ActionTypeCardInterface) => {
+                            return checkFeatureEnabledStatus(cardProps.identifier) && (
                                 <Card
                                     key={ cardProps.identifier }
-                                    className={ cardProps.disabled ? "action-card disabled" : "action-card" }
+                                    className={ cardProps.disabled ? "action-type disabled" : "action-type" }
```
[the remark this answers is not in the document]
Comment (ashanthamara) [reply features/admin.actions.v1/pages/actions.tsx:331]: Fixed by https://github.com/wso2/identity-apps/pull/6768

File: features/admin.actions.v1/pages/actions.tsx [hunk 4]
```diff
@@ -265,6 +266,51 @@ export const ActionTypesListingPage: FunctionComponent<ActionTypesListingPageInt
...
+                    <div className="action-types-grid-wrapper">
+                        <div className="action-types-grid">
+                            { cards }
+                        </div>
+                    </div>
+                </div>
+            );
+        }
+
+        return <>{ placeholders }</>;
```
[the remark this answers is not in the document]
Comment (ashanthamara) [reply features/admin.actions.v1/pages/actions.tsx:311]: Cant add the suggestion. it gives an error

**Pull request merged**